dangerfile.js

```diff
@@ -0,0 +1,26 @@
+const {fail} =  require("danger")
```

Comment:
@rafayet-monon Should we make this with this format `const { fail } =  require('danger')`?